extensions/kubeapi/rbac/rbac.go

```diff
@@ -66,3 +72,32 @@ var RoleTemplateGroupVersionResource = schema.GroupVersionResource{
 	Version:  Version,
 	Resource: "roletemplates",
 }
+
+// ProjectRoleTemplateBindingGroupVersionResource is the required Group Version Resource for accessing projectroletemplatebindings in a cluster, using the dynamic client.
+var ProjectRoleTemplateBindingGroupVersionResource = schema.GroupVersionResource{
+	Group:    GroupName,
+	Version:  Version,
+	Resource: "projectroletemplatebindings",
```

Comment:
Can we define this as a constant in rbac.go?

Reply:
I thought of that but this is used only once, we can leave this as is.